.github/workflows/test_comprehensive.yml

```diff
@@ -40,15 +40,15 @@ jobs:
           # test big sur on 3.9.0
           - python: 3.9
             platform: macos-11
-            toxenv: py39-macos-pyqt
+            toxenv: py39-macos-pyqt2
```

Comment:
I assume this is a typo.  (would need to be `pyqt5`, not `2`)
each item in that list needs to match one of the factors in the envlist at the top of tox.ini 

anyway, this is fixed in #4660  ... apologies for not seeing this PR